br/pkg/streamhelper/basic_lib_for_test.go

```diff
@@ -635,3 +644,84 @@ func (t *testEnv) unregisterTask() {
 		Name: "whole",
 	}
 }
+
+func (t *testEnv) ScanLocksInOneRegion(bo *tikv.Backoffer, key []byte, maxVersion uint64, limit uint32) ([]*txnlock.Lock, *tikv.KeyLocation, error) {
```

Comment:
Can we add a field like `LastLock` to the fakeRegion, so the lock info will keep consistency with the region info.

Reply:
Done, PTAL